ecs-cli/integ/cmd/compose.go

```diff
@@ -177,6 +176,7 @@ func testServiceHasAllRunningContainers(t *testing.T, p *Project, wantedNumOfCon
 
 	// Then
 	lines := strings.Split(string(out), "\n")
+	lines = lines[:len(lines)-3] // remove coverage metadata
```

Comment:
Dumb question: How come this is only needed in the compose command?

Minor nit: 3 feels like magic number